source/administration/devops-command-center.rst

```diff
@@ -18,16 +18,44 @@ User's Guide
 
 Incidents are events within Mattermost that are initiated to manage a specific situation/response in real-time. When the situation/response is complete, the incident is ended. Events that occurred within the incident can be reused in future incidents as playbooks and checklists.
 
+Using playbooks
+~~~~~~~~~~~~~~~~
+
+Playbooks are templates which are applied to an incident and define a group of tasks (stages) and steps to be followed in order to resolve the incident. Playbooks can be refined over a period of time to match the changing parameters of future related incidents. Incidents can only be started once they're associated with a playbook. It's not possible to start an incident from within the playbook backstage but you can create a playbook for use with a future incident.
+
+When you start an incident, you need to either create a playbook or select an existing one.
+
+**Creating a new playbook**
+
+The creator of a playbook is automatically added as the first member and can remove themselves once other members have been added. Only playbook members can use the playbook to start an incident - System Admins must be added to the member list to have access to a playbook.
+
+You can also create a new playbook outside of an incident. Select **+ New Playbook** to start a new playbook using either the blank template, or the **Incident Response Playbook** template provided.
+
+**Selecting an existing playbook**
+
+You can select any existing playbook that you have access to, from the drop-down list provided, when starting an incident.
+
+**Viewing playbooks**
+
+Open **Main Menu > Incidents & Playbooks > Playbooks** to view the playbooks associated with the current team and create a new one.
+
+Stages and steps
+~~~~~~~~~~~~~~~~~~
+
+When you create a playbook you can group your tasks into stages which have accompanying steps. Each step can be assigned a slash command. The steps within each stage must be completed in the order they're listed.
+
+Stages and steps are optional, and can be left empty by default. You can add stages and steps to a playbook that's being used in an active incident, but they won't be applied to the active incident.
```

Comment:
@agarciamontoro, I just tested this on a private playbook using the incident response template. Can you confirm that this is correct for all playbooks?

Reply:
@justinegeffen:

> The steps within each stage must be completed in the order they're listed.

This is not correct. You can complete the steps in the order you want.

> Stages and steps are optional, and can be left empty by default.

Not anymore! This was actually the cause of the bug you reported. After https://github.com/mattermost/mattermost-plugin-incident-response/pull/247, playbook must contain at least one stage with at least one step. In fact, if you add a stage with no steps, it will be filtered out and not added to the final playbook.

> You can add stages and steps to a playbook that's being used in an active incident, but they won't be applied to the active incident.

This is correct!